Update docs with admission controllers section

bernardjkim opened the pull request:
## Description
<!--Required. Provide high-level overview of what the change is for.-->
Add documentation on how to enable optional Kubernetes admission controllers in a custom `planet` image. Related changes in `planet` can be found in https://github.com/gravitational/planet/pull/809.

bernardjkim committed:
Update docs with admission controllers section

a-palchikov commented on docs/7.x/pack.md:
```diff
@@ -1117,3 +1117,21 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
+### Kubenetes Admission Controllers
+
+Optional Kubernetes admission controllers can be enabled when using a custom `planet`
+image. Create an environment file and specify a list of admission controllers to
```
This should mention that this is in addition to a set of predefined addmission controllers.

bernardjkim replied:
Good call. I'll include a sentence about the admissions controllers that are enabled by default.

knisbet commented on docs/7.x/pack.md:
```diff
@@ -1117,3 +1117,21 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
+
+### Kubenetes Admission Controllers
+
+Optional Kubernetes admission controllers can be enabled when using a custom `planet`
```
nitpick: I would link to custom system container, and use the same nomenclature as elsewhere in the docs which I think call planet the system container.

bernardjkim replied:
Yeah, that's probably a good idea to keep it consistent. I'll replaces references to planet to system container.

bernardjkim committed:
Address PR comments

- Include mention of predefined admission controllers.
- Replace references to planet with system container.
